src/converter/UserToStringConverter.java

```diff
@@ -0,0 +1,18 @@
+package converter;
+
+import model.User;
+
+import java.util.StringJoiner;
+
+public class UserToStringConverter {
+  private static final String USER_FIELDS_SEPARATOR = " ";
```

Comment:
You have the same duplicate in `StringToUserConverter`
Let's add `ConverterConstants` interface with such value